pkg/configuration/configuration.go

```diff
@@ -67,10 +86,38 @@ func initConfig() *Config {
 	return &c
 }
 
-func LoadConfig() *Config {
+func LoadConfig(cl client.Client) *Config {
+	getSecret(cl)
 	return initConfig()
 }
 
+// getSecret retrieves the host operator secret
+func getSecret(cl client.Client) error {
+	// get the secret name
+	secretName := os.Getenv("HOST_OPERATOR_SECRET_NAME")
+	if secretName == "" {
+		return nil
+	}
+
+	// get the secret
+	secret := &v1.Secret{}
+	namespacedName := types.NamespacedName{Namespace: defaultNamesapce, Name: secretName}
+	err := client.Client.Get(cl, context.TODO(), namespacedName, secret)
+	if err != nil {
+		return err
+	}
+
+	// get the domain and domain api key
+	mailgunDomain := secret.Data[mailgunDomain]
```

Comment:
The idea was to iterate over all keys/values and set them as `HOST_OPERATOR_<KEY> = <value>`.

Reply:
lol, :facepalm: sorry, fixed: 04f19047f465e80014d4a67dab49c7b68120b8e5